flax/linen/combinators.py

```diff
@@ -51,5 +51,8 @@ def __call__(self, *args, **kwargs):
 
     outputs = self.layers[0](*args, **kwargs)
     for layer in self.layers[1:]:
-      outputs = layer(outputs)
+      if isinstance(outputs, tuple):
```

Comment:
While we are at it, should we `**` expand `dicts` then?

Reply:
Addressed